OpenProjects.html

```diff
@@ -27,6 +27,7 @@
               <li><a href="#llvm_module_inliner">Evaluate and Expand the Module-Level Inliner</a></li>
               <li><a href="#llvm_undef_load">Remove undef: move uninitialized memory to poison</a></li>
               <li><a href="#llvm_abi_export">Add ABI/API export annotations to the LLVM build</a></li>
+              <li><a href="#llvm_lld_embedded">LLD Linker Improvements for Embedded Targets</a></li>
```

Comment:
why have you added to 2022 section?

Reply:
It was an oversight on my part. Moved the title to 2023 section.